yarn.lock

```diff
@@ -23,6 +23,13 @@
   dependencies:
     "@babel/highlight" "^7.10.4"
 
+"@babel/code-frame@^7.12.13", "@babel/code-frame@^7.18.6":
+  version "7.18.6"
+  resolved "https://artifacts.aue1e.internal:443/artifactory/api/npm/npm-okta-master/@babel/code-frame/-/code-frame-7.18.6.tgz#3b25d38c89600baa2dcc219edfa88a74eb2c427a"
```

Comment:
This is using a non-public artifactory link. Let's make sure this points to yarn's registry.

Reply:
Ah, I must've had the wrong `.npmrc` file in this directory. My bad. I'll fix that.

Reply:
f231d451b41fddc6dc3f2e420f58f33373259117

Reply:
There are a few ways to solve this, but I'd suggest partnering with other OSS repos like the Widget to work toward a common solution.

One promising option could be to use a custom [semgrep](https://oktawiki.atlassian.net/wiki/spaces/~875261636/pages/2300023474/Semgrep+in+GitHub+Actions) rule, connected to a GitHub Action to provide devs immediate feedback.